tfhe/src/shortint/server_key/tests/shortint.rs

```diff
@@ -3226,3 +3229,65 @@ where
     println!("(msg_true - msg_false) * control_bit  + msg_false = {clear_mux}, res = {dec_res}");
     assert_eq!(clear_mux, dec_res);
 }
+
+fn shortint_trivial_pbs<P>(param: P)
+where
+    P: Into<PBSParameters>,
+{
+    let param = param.into();
+    let full_modulus = param.message_modulus().0 as u64 * param.carry_modulus().0 as u64;
+    let keys = KEY_CACHE.get_from_param(param);
+    let (cks, sks) = (keys.client_key(), keys.server_key());
+
+    let check_trivial_bootstrap = |clear, lut: &LookupTableOwned| {
+        let trivial_ct = sks.unchecked_create_trivial(clear);
+        let non_trivial_ct = cks.unchecked_encrypt(clear);
+
+        let trivial_res = sks.apply_lookup_table(&trivial_ct, lut);
+        let non_trivial_res = sks.apply_lookup_table(&non_trivial_ct, lut);
+        assert_eq!(trivial_res.noise_level(), NoiseLevel::ZERO);
```

Comment:
we can add the check regardless, since the input has to be a trivial, the input mask has to be zero, and since output is also a trivial its mask must be zero (and the impl do no touches the mask)